include/deal.II/base/symmetric_tensor.h

```diff
@@ -539,7 +539,7 @@ class SymmetricTensor
   /**
    * Default constructor. Creates a tensor with all entries equal to zero.
    */
-  SymmetricTensor () = default;
```

Comment:
~~why did you remove `= default` keyword?~~ ah, i see below that you need explicitly initialize to zero AD numbers.